amoro-web/src/views/tables/components/Optimizing.vue

```diff
@@ -85,16 +86,31 @@ const sourceData = reactive({
   table: '',
   ...query,
 })
+const actionType = ref<ILableAndValue>()
+const actionTypeList = ref<ILableAndValue[]>([])
+const statusType = ref<ILableAndValue>()
+const statusTypeList = ref<ILableAndValue[]>([])
+
+async function getQueryDataDictList() {
+  const tableProcessTypes = await getTableOptimizingTypes({...sourceData})
+  const typesList = Object.entries(tableProcessTypes).map(([typeName, displayName]) => ({ label: displayName as string, value: typeName}))
+  const status = Object.entries(statusMap).map(([key, value]) => ({ label: value.title , value: key }));
+
+  actionTypeList.value = typesList
+  statusTypeList.value = status
+}
```

Comment:
Maybe need add some mock data in the front-end mock.

Reply:
Fixed